<launch>

    <arg name="pc_topic" default="/head_camera/depth_registered/points" />

    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" />
    <include file="$(find applications)/launch/navigation.launch" />
    <include file="$(find fetch_api)/launch/ar_desktop.launch">
        <arg name="cam_image_topic" value="$(arg pc_topic)" />
    </include>
    <include file="$(find fetch_api)/launch/move_group.launch" />
    <node pkg="applications" type="gripper_im_server.py" name="gripper_im_server" output="screen" />
    <node pkg="shuguru" type="command_server.py" name="command_server" output="screen"/>
    <node pkg="shuguru" type="manipulation_server.py" name="manipulation_server" output="screen"/>
    <node pkg="shuguru" type="state_machine.py" name="state_machine" output="screen"/>

</launch>
